.github/workflows/build.yml

```
@@ -36,6 +36,7 @@ jobs:
         uses: actions/upload-artifact@v3
         with:
           name: build-output-ubuntu
+          retention-days: ${{ fromJSON('{"false":90,"true":3}')[github.event_name == 'pull_request'] }}
```

Does GitHub expression language not have a ternary?

https://docs.github.com/en/actions/learn-github-actions/expressions#operators

What I guess not? Should `(a && b) || c` work? I guess in this case

```suggestion
          retention-days: ${{ github.event_name == 'pull_request' && 3 || 90 }}
```

seems clearer to me but IDK whatever if it works it works both are a little obscure